foia_hub/static/js/main.js

```diff
@@ -29,6 +29,7 @@ Utils = {
   }
 };
 
+var times_toggled = 0;
```

Comment:
Do we need this to be a global? Can we instead define it within the `$(document).ready()`?